ts/editor/NoteEditor.svelte

```diff
@@ -603,4 +729,37 @@ the AddCards dialog) should be implemented in the user of this component.
         flex-direction: column;
         height: 100%;
     }
+
+    :global(.image-occlusion) {
+        position: fixed;
+    }
+
+    :global(.image-occlusion .tab-buttons) {
+        display: none !important;
+    }
+
+    :global(.image-occlusion .top-tool-bar-container) {
+        margin-left: 28px !important;
+    }
+    :global(.top-tool-bar-container .icon-button) {
+        height: 36px !important;
+    }
+    :global(.image-occlusion .tool-bar-container) {
+        top: unset !important;
+        margin-top: 2px !important;
+    }
+
+    :global(.active-io-btn) {
+        background: var(--button-primary-bg) !important;
+        color: white !important;
+    }
```

Comment:
This style belongs in the svelte file that uses it, and then doesn't need to be global. Maybe the same applies to some of the others too?

Reply:
`Unused CSS selector ".active-tool"svelte(css-unused-selector)`

But class needed for hiding the element, so HTML element does not need global but component needs global.

Reply:
Yep, because of https://github.com/ankitects/anki/pull/2485#discussion_r1271678478, I see this will need to stay global.